dockercli.md

```diff
@@ -164,7 +163,7 @@ There is also a [Minio Client Docker image](https://hub.docker.com/r/minio/minio
 that you can use as well but it will be more difficult to use with the following scripts due
 to container isolation. 
 
-To man an entry in the mc config use:
+To man an entry in the mc config, use:
```

Comment:
should 'man' be 'make' here?

Reply:
Good catch. I haven't been using `mc` so it didn't look as out of place to me.